### DISCUSSION TITLE
Resizing the nodes from all four corners instead of bottom right

### DISCUSSION THREAD

**Discussion by kayselmecnun** [Comfy-Org/ComfyUI_frontend#176] [Ideas]
Resizing the nodes from all four corners instead of bottom right would be a good UX, right now trying to aim a tiny area inside of a node cluster a bit hard. Even there might be a tiny icon that represents the expanding

**Comment by SineSwiper** [comment]
This problem makes it complicated to create groups on already established nodes (which is all of them for me):

1. Create the group above the nodes I want to capture them in.
2. Resize the group with the lower-right corner to include all of the nodes I want.
3. Despair, as now, I can't resize on the upper-left corner to get rid of the dead space.  And if I move the group, it now moves everything.
 
Also, if I start the group really close to the upper-left spot where I want it, the always-on-bottom nature of groups ensures that I can never locate the lower-right corner, because the nodes are blocking the way!

**Reply by fnf** [reply]
It is possible to move the group only (without moving the nodes within) as a workaround: you need to click and hold the title of the group, then press Ctrl, then start dragging.

A direct solution in case you just want to fit the group over its node is to use the "Fit Group to Nodes" feature (right-clicking on the group's title).

**Comment by Zandebar** [comment]
I agree with OP, I've hit that and sometimes its easier to adjust from all four corners.
